components/list/list-item-drag-drop-mixin.js

```diff
@@ -105,6 +105,7 @@ class DragState {
 
 	_cleanUpOnLeave() {
 		if (!this._activeDropTarget) return;
+		this._activeDropTarget._draggingOver = false;
```

Comment:
I moved this a bit from where we had it on our call (in `setActiveDropTarget`). Seemed more appropriate here!

Reply:
Yeah this spot seems perfect!